calibration/PARADM/CGINS-PARADM-50164__20130304.csv

```diff
@@ -1,2 +1,2 @@
 serial,name,value,notes
-50164,CC_bsipar_par_scaling,1.00E-05,
+50164,CC_bsipar_par_scaling,1.00E-06,

```

Comment:
If we are changing this calibration value, then I need to update the Metadata Change Log. Are there more of these?

Reply:
No, just the one.  I grepped the PARADM directory and that looked to be the only one with 1e-5.
Also I don't know anything about Metadata Change Logs.  Lets chat about that.